### PR TITLE
Unify rendering PanelBar

### PR DESCRIPTION
**Pull request by zhpenkov**
closes: https://github.com/telerik/kendo-themes-private/issues/215

### CODE REVIEW

**Review by JoomFX — APPROVED** [review]
If the item icon from the visual tests is removed on purpose, then everything looks great! :)

NOTE: When the Spacing module is merged, some of the variable values will need to be updated.

**Comment by zhpenkov** [comment]
> If the item icon from the visual tests is removed on purpose, then everything looks great! :)
> 
> NOTE: When the Spacing module is merged, some of the variable values will need to be updated.

The item icon was actually missing in the rtl test, so I added it.

**Comment by Juveniel** [comment]
@zhpenkov please mark the changed styles commit accordingly e.g `feat(panelbar): update styles to match universal rendering`

File: packages/html/src/panelbar/panelbar-item.tsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+                    ["k-panelbar-header"]: header,
+                    [`k-level-${level}`]: level || level === 0,
+                    ["k-expanded"]: expanded
+                }
+            )}
+        >
+            <span
+                className={classNames(
+                    "k-link",
+                    stateClassNames("k-link", {
```
Comment (inikolova) [inline packages/html/src/panelbar/panelbar-item.tsx:68]: "k-link" is duplicated?

Comment (zhpenkov) [reply packages/html/src/panelbar/panelbar-item.tsx:68]: The `stateClassNames` function accepts 2 parameters, `_componentClassName`, and `props`, that's why `k-link` is passed inside and used for a second time. There is no extra `k-link` class added to this element.

**Review by inikolova — APPROVED** [review]

**Pull request merged**